routes/stations.js

```diff
@@ -18,6 +18,18 @@ router.get('/:_id', function(req, res, next) {
   });
 });
 
+/* GET stations given network _id */
+router.get('/network/:network_id', function(req, res, next) {
+  Station.find({"network_id": mongoose.Types.ObjectId(req.params.network_id)}, function(err, station) {
+    if(err) {
+      console.log(err);
```

Comment:
Doesn't this return many stations?
If this returns many stations, it should be names `stations` in the return value.